shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/metadata/node/PipelineMetaDataNode.java

```diff
@@ -34,6 +34,11 @@ public final class PipelineMetaDataNode {
     
     public static final Pattern BARRIER_PATTERN = Pattern.compile(JOB_PATTERN_PREFIX + "/barrier/(enable|disable)/\\d+");
     
+    private static final String MIGRATION_JOB_PATTERN_PREFIX = DataPipelineConstants.DATA_PIPELINE_ROOT + "/jobs/(j01[0-9a-f]+)";
+    
+    public static final Pattern MIGRATION_JOB_CONFIG_PATTERN = Pattern.compile(MIGRATION_JOB_PATTERN_PREFIX + "/config");
+    
```

Comment:
1, It's not just used for MIGRATION, so prefix is not necessary;

2, `/jobs/(j01[0-9a-f]+`, 1) `01` should not be hard-coded, 2) it's better to reuse one prefix, the old one could be updated if it's wrong;